src/layers/recurrent.jl

```diff
@@ -67,6 +81,12 @@ end
 
 flip(f, xs) = reverse(f.(reverse(xs)))
 
+function (m::Recur)(x::AbstractArray{T, 3}) where T
+  h = [m(x[:, :, i]) for i in 1:size(x, 3)]
```

Comment:
```suggestion
  h = [m(@view(x[:, :, i])) for i in 1:size(x, 3)]
```
cc @ToucheSir

Reply:
Since:
```julia
julia> @macroexpand @view x[:, :, 1]
:(true && (view)(x, :, :, 1))
```
I'd favour:
```suggestion
  h = [m(view(x, :, :, i)) for i in 1:size(x, 3)]
```